tests/pact/bad/bad-import-unimported-reference.repl

```diff
@@ -16,3 +16,5 @@
   (defcap G () true)
   (defun g () (f))
 )
+
```

Comment:
Hmm, what does this addition do? Presumably this test was already failing so it never gets to this code. If you need a new "bad test" you have to make another `.repl` file